components/PageHeader.vue

```diff
@@ -58,6 +75,69 @@
         type: Boolean,
         default: false
       }
+    },
+
+    data() {
+      return {
+        scrollPosition: 0,
+        windowWidth: 992,
+        desktopWidth: 992
```

Comment:
Is there a better way to get this from our css breakpoint in case we change the specific value?

Reply:
it should remain the same value as long as we keep using bootstrap since that's the standard desktop breakpoint

Reply:
we can export the value from sass variables 
```
:export{
    $bplarge: 992px;
}
```
then import
`import { $bplarge } from '../assets/scss/variables.scss';`
but for some reason this value isn't actually picked up by the app when setting:
`desktopWidth: $bplarge`

I think it's very unlikely the value will change often, 992 is the default desktop breakpoint for bootstrap so hardcoding isn't a problem